mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/GroupRegistration.java

```diff
@@ -80,10 +83,39 @@ String asString() {
         }
     }
 
+    static class RetryExchangeName {
```

Comment:
this duplicate some methods to `WorkQueueName`. But Keeping different class names helps to take advance of strong typing